bigbluebutton-html5/private/locales/en.json

```diff
@@ -398,6 +398,8 @@
     "app.audioModal.settingsTitle": "Change your audio settings",
     "app.audioModal.helpTitle": "There was an issue with your media devices",
     "app.audioModal.helpText": "Did you give permission for access to your microphone?  Note that a dialog should appear when you try to join audio, asking for your media device permissions, please accept that in order to join the audio conference.  If that is not the case, try changing your microphone permissions in your browser's settings.",
+    "app.audioModal.help.noSSL": "The webpage must use HTTPS for microphone access to work. Please contact the administrator of your website",
```

Comment:
"This page is unsecured. For microphone access to be allowed the page must be served over HTTPS. Please contact the server administrator."